config/users.yml

```diff
@@ -1,4 +1,24 @@
 ---
+
+# This file configures platform-wide users.
+# - All users in this file get access to Grafana.
+# - Users with `cf_admin: true` get full admin privileges in Cloud Foundry.
+# - Users without the `cf_admin` option or where it is false get
+#   Global Auditor, which allows them to see everything except secrets in
+#   app environment variables.
+#
+# HOW TO ADD NEW USERS
+# - This file requires users to sign in using Google SSO. To do that we need
+#   their Google ID, which is a large number.
+# - If a new user is configured in this file who didn't already exist, they
+#   lose Global Auditor when they first log in. Subsequent pipeline runs will
+#   permanently re-add it. This seems to be UAA behaviour.
```

Comment:
We discovered the source of this weirdness and fixed it in #2185. These instructions are still good as they tell us the Google ID, but we can remove this warning. I've set a reminder to edit them later :)